openapi/components/schemas/GatewayAccount.yaml

```
@@ -370,10 +370,13 @@ properties:
         Does nothing except return an approved `setup` transaction.
         This is the default behavior.
   readyToPayoutInstruction:
-    type: string
+    type:
+      - 'string'
+      - 'null'
     default: none
     description: |-
-      Defines a mode of gateway payout behaviour.
+      Defines a mode of gateway payout behaviour for [StorefrontReadyToPayout](https://all-rebilly.redoc.ly/tag/Purchases/operation/StorefrontPostReadyToPayout).
+      If a `null`, a gateway does not support payout operation.
```

suggestion: for readability 
```suggestion
      If this value is `null`, payouts are not supported by the gateway.
```